public/locales/pt/common.json

```
@@ -0,0 +1,15 @@
+{
+  "title": "O modo mais fácil de criar um",
+  "readme": "README",
+  "description": "Nosso simples editor permite que você facilmente adicione e customize todas as seções que você precisa para o readme do seu projeto",
+  "get-started": "Comece aqui",
+  "100-free": "100% Gratuito",
+  "made-with-love": "Feito com ",
+  "by": " por",
+  "language": "Idioma",
+  "english": "Inglês",
+  "french": "Francês",
+  "portuguese": "Português",
+  "spanish": "Espanhol",
+  "translation-error": "Tradução Incorreta?"
```

Same here, no need to capitalize:

```suggestion
  "translation-error": "Tradução incorreta?"
```